### ISSUE TITLE
"Getting Started" is unclear

### ISSUE THREAD

**Issue by NYCJames** [glowing-bear/glowing-bear#1137]
In the getting started section, it tells the user to use "certbot certonly --standalone -d **localhost**" which will output 
![image](https://user-images.githubusercontent.com/38173610/83361434-f750bd00-a356-11ea-8f83-706f3dffe648.png)

This is because in connection settings, the WeeChat relay hostname is prepopulated to be localhost and the site uses whatever is in the field to give the user the command to input.

The same is true for the input "cat /etc/letsencrypt/live/**localhost**/{fullchain,privkey}.pem > ~username/.weechat/ssl/relay.pem"

**Comment by lorenzhs** [comment]
Yeah we should probably make a note about that, thanks for your report
